src/common/src/system_param.rs

```diff
@@ -0,0 +1,154 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::HashSet;
+
+use risingwave_pb::meta::SystemParams;
+
+use crate::error::{ErrorCode, RwError};
+
+pub const BARRIER_INTERVAL_MS_KEY: &str = "barrier_interval_ms";
+pub const CHECKPOINT_FREQUENCY_KEY: &str = "checkpoint_interval";
```

Comment:
Looks much better now! BTW, if you don't mind always using the same name for the `const`, proto's field name, and the etcd key, we may use the `paste` to convert the case.

https://docs.rs/paste/latest/paste/#case-conversion